web/settings.py

```diff
@@ -47,6 +47,9 @@ def get_secret(secret_name, region_name):
 
 hosts_list = os.getenv("DOMAIN", "localhost:brain-score-web-dev.us-east-2.elasticbeanstalk.com").split(":")
 hosts_list.append("Brain-score-web-prod-updated.kmk2mcntkw.us-east-2.elasticbeanstalk.com")  # updated prod env
+
+if os.getenv("DJANGO_ENV") == "development": hosts_list.append("127.0.0.1")
```

Comment:
could also repurpose the `DEBUG` flag? https://docs.djangoproject.com/en/4.2/ref/settings/#debug

Reply:
Sure, I can make that change if you'd prefer it that way. 👍  My reasoning was to decouple local dev from other environments where you might want to turn on debug logging in the future like staging or testing environments. LMK which you'd prefer

Reply:
One other option that occurred to me is that the debug flag could be toggled on by default if `DJANGO_ENV=development` and then it's still possible to use the debug flag in other environments but it would preclude having to pass multiple env vars in on the command line. Let me know which option you guys prefer

Reply:
no strong preference from my side, @kvfairchild @mike-ferguson please chime in if you have thoughts

Reply:
I am leaning more towards the original solution, because we like @fairchildseb says there could be a case where we want to use DEBUG and not DJANGO_ENV, or vice versa, and I feel like the decoupling is better here. The original change in the PR still seems like the ideal solution to me